modules/accessing-metrics-outside-cluster.adoc

```diff
@@ -84,5 +84,7 @@ The output shows the status for each application pod that Prometheus is scraping
 +
 [NOTE]
 ====
-The formatted example output uses a filtering tool, such as `jq`, to provide the formatted indented JSON. See the link:https://stedolan.github.io/jq/manual/[jq Manual] for more information about using `jq`.
+* The formatted example output uses a filtering tool, such as `jq`, to provide the formatted indented JSON. See the link:https://stedolan.github.io/jq/manual/[jq Manual] for more information about using `jq`.
```

Comment:
Out of scope to your work, so it's up to you if you want to do this. But per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute), links to external websites should typically be followed with the destination in parentheses:
```suggestion
* The formatted example output uses a filtering tool, such as `jq`, to provide the formatted indented JSON. See the link:https://stedolan.github.io/jq/manual/[jq Manual] (jq documentation) for more information about using `jq`.
```